packages/e2e-tests/src/specs/editor/taxonomy.js

```diff
@@ -49,41 +49,56 @@ async function goToAndExpandTaxonomyPanel() {
 
 async function addCategory(name, parent) {
   await expect(page).toClick('button', { text: 'Add New Category' });
-  await expect(page).toFill('input[name="New Category Name"]', name);
 
   if (parent) {
     await expect(page).toClick('button[aria-label="Parent Category"]');
     await expect(page).toMatchElement('li[role="option"]', { text: parent });
-    await expect(page).toClick('li[role="option"]', { text: parent });
-    await expect(page).toMatchElement(
-      'li[role="option"][aria-selected="true"]',
-      { text: parent }
-    );
 
+    await expect(page).toClick('li[role="option"]', { text: parent });
     await expect(page).toMatchElement('button[aria-label="Parent Category"]', {
       text: parent,
```

Comment:
So this was having trouble determining that the dropdown was closed and submit button available, what I did was move it so that if it's a nested category do the dropdown category first and then add your input value and hit enter - eliminates the need to get puppeteer to see the button.